.github/workflows/build.yml

```diff
@@ -76,9 +75,14 @@ jobs:
         path: "*.log"
     - name: Report check status
       run: exit $(< make-check.status)
+    - name: Check expected plugins
+      run: |
+        if [[ -e ".expected-plugins/${{ matrix.container_tag }}" ]]; then
+          /checks/check-built-plugins.sh ".expected-plugins/${{ matrix.container_tag }}"
+        fi
```

Comment:
For some containers, no plugin checks are done?

Reply:
`.expected-plugins/` does not yet exist in the `main` branch. If you want, we can adopt the CirrusCI behavior for GitHub Actions, but I'd like to make that change in `collectd-6.0` first and then cherry-pick to here, so that the PR remains a "pure" import (i.e. no novel changes).